winlogbeat/docs/winlogbeat-filtering.asciidoc

```diff
@@ -0,0 +1,17 @@
+[[filtering-data]]
+=== Filtering the Exported Data
+
+include::../../libbeat/docs/filtering.asciidoc[]
+
+For example, the following filter configuration drops the `???` field from the exported document.
+
+// Can you help me provide a realistic example here?
```

Comment:
Here's an example for Winlogbeat. It just drops a few fields that are rarely used (provider_guid, process_id, thread_id, version) and one nested field, `event_data.ErrorSourceTable`.
```
filter:
  - drop_fields:
      fields: [provider_guid, process_id, thread_id, version, event_data.ErrorSourceTable]
```